src/Private/Helpers.ps1

```diff
@@ -954,7 +942,7 @@ function Remove-PodeEmptyItemsFromArray {
         return @()
     }
 
-    return @(@($Array -ne ([string]::Empty)) -ne $null)
+    return @($null -ne @($Array -ne ([string]::Empty)))
```

Comment:
This needs to be flipped back to `return @(@($Array -ne ([string]::Empty)) -ne $null)`, the `$null` is intentionally on the right-hand side to filter out nulls from the array.

Reply:
ok

Reply:
I changed because Visual Studio is bothering me with
$null should be on the left side of the equality comparisons.

Reply:
Yeah, my VSCode does the same, it's annoying. Normally the null should be on the left-hand side when just doing a standard null check, but for arrays it's perfectly valid on the right-hand side to filter out nulls - I will IDEs understood that, haha.